module/VuFindAdmin/src/VuFindAdmin/Controller/TagsController.php

```diff
@@ -198,18 +198,18 @@ protected function getConfirmDeleteMessages($count)
         $user = $this->getTable('user')
             ->select(['id' => $this->getParam('user_id')])
             ->current();
-        $userMsg = (false !== $user)
+        $userMsg = is_object($user)
             ? $user->username . " (" . $user->id . ")" : "All";
```

Comment:
"All" here doesn't have a leading space while the others do. Any reason for that?